.github/workflows/build.yml

```diff
@@ -16,6 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: gradle/wrapper-validation-action@v1
```

Comment:
Using name field would be nice. wdyt?
```
steps:
  name: Validation step
  uses: ...
```
Same for other changes :)

Reply:
Fixed.